traits/tests/test_file.py

```diff
@@ -9,7 +9,10 @@
 # Thanks for using Enthought open source!
 
 import os
+import sys
+
 import unittest
+from pathlib import Path
```

Comment:
Let's follow the usual Enthought rules for import order here: group all std. lib. imports together (so lose the blank line between `sys` and `unittest`, and keep the modules in alphabetical order.